electrum/plugins/jade/jade.py

```diff
@@ -0,0 +1,458 @@
+import os
+import base64
+import semver
```

Comment:
No problem.  In some of our code we use 'semver' as it handles eg. beta and rc versions - but yes since we don't really expect those versions to be in use out in the wild the more straightforward check you suggest should not cause any issues - and as you say it removes a dependency.